jubilant/_juju.py

```diff
@@ -313,6 +318,7 @@ def _cli(
                 capture_output=True,
                 encoding='utf-8',
                 input=stdin,
+                timeout=timeout,
```

Comment:
It feels a bit odd that we wrap `CalledProcessError` but don't wrap `TimeoutExpired`. I guess the difference is that there's no public way to provide the timeout, but it does feel a bit off.

Reply:
Yeah, I am happy wrapping it if/when we make `timeout` public.